src/test/regress/sql/with_clause.sql

```diff
@@ -3,6 +3,9 @@
 -- m/ERROR:  Too much references to non-SELECT CTE \(allpaths\.c:\d+\)/
 -- s/\d+/XXX/g
 --
+-- m/ERROR:  Cannot broadcast Replicated locus \(cdbllize\.c:\d+\)/
+-- s/\d+/XXX/g
+--
```

Comment:
I think we miss some tests here or the result of existed tests should be changed.